### PR TITLE
Adds further information about proxy settings on the scripts

### PR DESCRIPTION
**Pull request by camilamacedo86**
#### What this PR does / why we need it:

For we have further information when/if we need troubleshoot

#### Which issue(s) this PR fixes:
Motivates by # [sc-76762]

#### Special notes for your reviewer:
Just add a few extra logs

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: scripts/common/proxy.sh [hunk 1]
```diff
@@ -1,17 +1,21 @@
 
 function proxy_bootstrap() {
     if [ -n "$HTTP_PROXY" ]; then
+        log "HTTP proxy has the value: $HTTP_PROXY"
```
Comment (emosbaugh) [inline scripts/common/proxy.sh:4]: there is a better message 3 lines below (line 7 in this case) each of these log messages.

Comment (camilamacedo86) [reply scripts/common/proxy.sh:4]: Yes, you are 100% right

File: scripts/common/proxy.sh [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
         ENV_PROXY_ADDRESS="$HTTP_PROXY"
         export https_proxy="$HTTP_PROXY"
         printf "The installer will use the proxy at '%s' (imported from env var 'HTTP_PROXY')\n" "$ENV_PROXY_ADDRESS"
     elif [ -n "$http_proxy" ]; then
+        log "HTTP proxy has the value: $http_proxy"
```
Comment (camilamacedo86) [inline scripts/common/proxy.sh:9]:
```suggestion
```

File: scripts/common/proxy.sh [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
         ENV_PROXY_ADDRESS="$http_proxy"
         export https_proxy="$http_proxy"
         printf "The installer will use the proxy at '%s' (imported from env var 'http_proxy')\n" "$ENV_PROXY_ADDRESS"
     elif [ -n "$HTTPS_PROXY" ]; then
+        log "HTTP proxy has the value: $HTTPS_PROXY"
```
Comment (camilamacedo86) [inline scripts/common/proxy.sh:14]:
```suggestion
```

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**